src/app/space/add-app-overlay/add-app-overlay.component.spec.ts

```diff
@@ -327,6 +327,35 @@ describe('AddAppOverlayComponent', () => {
       });
     });
 
+    it('continue button should be disabled on navigation in progress', async(() => {
```

Comment:
Please pass `DoneFn` and call `done()` at the end of `fixture.whenStable().then(() => {...}`. Otherwise tests will always pass.

Reply:
I think ```DoneFn``` is not required as I am already doing it in ```async```.

Reply:
Right, missed that `async` wrapper here